packages/lesswrong/components/search/SearchFilters.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { MutableRefObject } from 'react';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import { RefinementListExposed, RefinementListProvided } from 'react-instantsearch/connectors';
+import { AlgoliaIndexCollectionName } from '../../lib/search/algoliaUtil';
+import { ToggleRefinement, NumericMenu, ClearRefinements, connectRefinementList } from 'react-instantsearch-dom';
+import { forumTypeSetting, taggingNamePluralSetting } from '../../lib/instanceSettings';
+import { Link } from '../../lib/reactRouterWrapper';
+import Select from '@material-ui/core/Select';
+import {
+    ElasticSorting,
```

Comment:
```suggestion
  ElasticSorting,
```

Mis-matched indentation here